tests/test_open.py

```diff
@@ -83,3 +83,39 @@ def test_open(contiguous, urlpath, cparams, dparams, nchunks, chunk_nitems, dtyp
 def test_open_fake():
     with pytest.raises(FileNotFoundError):
         _ = blosc2.open("none.b2nd")
+
+
+@pytest.mark.parametrize("offset", [0, 42])
+@pytest.mark.parametrize("urlpath", ["schunk.b2frame"])
+def test_open_offset(offset, urlpath):
+    urlpath_temp = urlpath + ".temp"
+
+    blosc2.remove_urlpath(urlpath)
+    blosc2.remove_urlpath(urlpath_temp)
+
+    # Create a temporary file with data.
+    data = np.arange(100)
+    schunk_temp = blosc2.SChunk(data=data, urlpath=urlpath_temp)
+    del schunk_temp
```

Comment:
Maybe an idiom that you can use here is `_ = blosc2.SChunk(data=data, urlpath=urlpath_temp)` that would remove the need of the additional `del`?

Reply:
Thanks! See comment below about unneeded references.